zerver/views/documentation.py

```diff
@@ -19,6 +20,14 @@
 from zerver.openapi.openapi import get_endpoint_from_operationid, get_openapi_summary
 
 
+@dataclass
+class ArticlePath:
+    article: str
+    article_http_status: int
+    endpoint_name: str
+    endpoint_method: str
```

Comment:
I kinda feel like this name isn't quite right; maybe we want:
```
class DocumentationArticle:
    article_path: str
    article_http_status: int
    endpoint_path: Optional[str]  # should be `None` rather than `""` if not a REST API page?
    endpoint_method: Optional[str]  # should be `None` rather than `""` if not a REST API page?
```

(I think `endpoint_path`, since `endpoint_name` would be the `operationId`, not the thing with `/`s in it).

Reply:
Changed the naming.